### PR TITLE
DOCSP-35980: Insert One usage example

### PR DESCRIPTION
**Pull request by norareidy**
Adds a usage example showing how to insert a document.
JIRA - https://jira.mongodb.org/browse/DOCSP-35980
Staging - https://preview-mongodbnorareidy.gatsbyjs.io/laravel/DOCSP-35980-staging/usage-examples/insertOne/

### Checklist

- [ ] Add tests and ensure they pass
- [ ] Add an entry to the CHANGELOG.md file
- [ ] Update documentation for new features

### CODE REVIEW

File: docs/includes/usage-examples/InsertOneTest.php
Comment (GromNaN) [inline docs/includes/usage-examples/InsertOneTest.php:27]: This example could be completed by the more expressive form (if anyone need to update the `$movie` model before saving):

```php
$movie = new Movie([
    'title' => 'Marriage Story',
    'year' => 2019,
    'runtime' => 136,
]);
$movie->save();
```

File: docs/usage-examples/insertOne.txt
Comment (rustagir) [inline docs/usage-examples/insertOne.txt]:
```suggestion
You can insert a document into a collection by calling the ``create()`` method on
an Eloquent model or query builder.
```

File: docs/usage-examples/insertOne.txt
Comment (rustagir) [inline docs/usage-examples/insertOne.txt]:
```suggestion
To learn how to edit your Laravel application to run the usage example, see the
:ref:`Usage Example landing page <laravel-usage-examples>`.
```

File: docs/usage-examples/insertOne.txt
Comment (rustagir) [inline docs/usage-examples/insertOne.txt:70]: Q: will this be merged only after write operations is merged? Link is broken as of now

**Review by rustagir — APPROVED** [review]
lgtm with a few comments!

File: docs/usage-examples/insertOne.txt
Comment (norareidy) [reply docs/usage-examples/insertOne.txt:70]: Yeah I'll wait until after write operations is merged

File: docs/usage-examples/insertOne.txt
Comment (GromNaN) [inline docs/usage-examples/insertOne.txt]: It seems to me that this 2 sentences tell the same thing. 
```suggestion
You can insert a document into a collection by calling the ``create()`` method on
an Eloquent model, specifying the values for each field.
```

File: docs/usage-examples/insertOne.txt
Comment (GromNaN) [inline docs/usage-examples/insertOne.txt]: This is valid JSON. Can you format with a language: "json", "js" or "ts"?

File: docs/includes/usage-examples/InsertOneTest.php
Comment (ccho-mongodb) [reply docs/includes/usage-examples/InsertOneTest.php:27]: @GromNaN Would you recommend replacing the ``create()`` example with one that shows ``save()``? If ``save()`` seems more common and helpful, we could update the example to show that method of inserting a single document instead.

Comment (GromNaN) [reply docs/includes/usage-examples/InsertOneTest.php:27]: No, `Movie::create([...])` is the most common way to create a model.

There is also `Movie::createOrFirst()` that can be used (using `findOneAndUpdate` with upsert)

**Review by GromNaN — APPROVED** [review]
LGTM as this.

**Pull request merged**
